projects/ngrx.io/content/guide/store/recipes/custom-elements.md

```diff
@@ -0,0 +1,65 @@
+# Using Store with Angular Elements
+
+## Creating Angular Elements
+
+Using Store with Angular Elements is similar to using it with standard Angular Component.
+
+## Tutorial
+
+The following tutorial shows you how to manage the state of a counter, and how to select and display it within an Angular Elmements and use the same within an Angular Project and an static page. Try the <live-example name="store-elements" downloadOnly>Anguler Elements Project</live-example>.
+
+1. Create a new application by running the below command
+```sh
+ng new ngRxElementDemo --createApplication=false
+```
+2. Create a new Angular Library using below command
+```sh
+ng g library my-counter
+```
+3. Add code mentioned in [Getting Started Guide](guide/store#installation) to new created.
+4. Rename `app.module.ts` to `counter.module.ts` file and add the below code.
+<code-example header="src/lib/counter.module.ts" path="store-elements/projects/my-counter/src/lib/app/counter.module.ts">
+</code-example>
+5. Add 2 application within the project using the below command
+```sh
+ng g application elementApp
+ng g application counterelement
+```
+6. Within elementApp add the below code in `app.module.ts`.
+<code-example header="elementApp/src/app.module.ts" path="store-elements/projects/elementApp/src/app.module.ts">
+</code-example>
+7. Add the below code in `app.component.html`.
+<code-example header="elementApp/src/app.component.html" path="store-elements/projects/elementApp/src/app.component.html">
+</code-example>
+8. Run the below command and import the code given in `polyfills.ts` file available in both the app created.
+```sh
+npm i @webcomponents/webcomponentsjs -save
+import '@webcomponents/webcomponentsjs/custom-elements-es5-adapter.js';
+```
+8. Verify the app by runnig it locally using below command.
+```sh
+ng build my-counter && ng serve — project=elementApp -o
+```
+9. For building an custom-element.js file which can be used outside an Angular App. Go to `counterelement` App and add the below code in `app.module.ts`.
+<code-example header="counterelement/src/app.module.ts" path="store-elements/projects/counterelement/src/app.module.ts">
+</code-example>
+10. Run the below commands to install a package to merge files to create `custom-element.js` file.
+```sh
+npm install jscat --save-dev
+```
+11. Add a new folder `htmlapp` in root, create a new file in `index.html` and add the below code.
+<code-example header="htmlapp/index.html" path="store-elements/htmlapp/index.html">
+</code-example>
+12. Add the below command to `package.json`.
+```sh
+"build-element": "ng build my-counter && ng build --project=counterelement --prod --output-hashing=none",
+"package": "jscat ./dist/counterelement/runtime.js ./dist/counterelement/polyfills.js ./dist/counterelement/scripts.js ./dist/counterelement/main.js > htmlapp/counter-element.js",
+```
+13. Run the below command to create and run the html app.
```

Comment:
Consider wording like...Run the “following commands to build and package your custom elements into a single “js” file output.”